qa/tasks/ceph_manager.py

```diff
@@ -1803,7 +1804,7 @@ def create_pool(self, pool_name, pg_num=16,
         :param erasure_code_use_overwrites: if true, allow overwrites
         """
         with self.lock:
-            assert isinstance(pool_name, basestring)
+            assert isinstance(pool_name, six.string_types)
```

Comment:
Interesting, wouldn't it be better if just use:
```
from six import string_types as basestring
```
The advantage is to minimize diffs, and postpone merge conflicts to the later times?

Reply:
I prefer to not hide the python2 builtin `basestring` and use six.string_types explicit here.